### PR TITLE
optimization

### PR DESCRIPTION
**Pull request by effine**
>  optimization .gitignore file
> delete warning of program

### CODE REVIEW

File: src/main/java/com/dongxuexidu/douban4j/model/app/AccessToken.java [hunk 1]
```diff
@@ -7,81 +7,88 @@
...
-  
-  public AccessToken (String accessToken, int expiresIn, String refreshToken, String doubanUserId) {
-    this.accessToken = accessToken;
-    this.doubanUserId = doubanUserId;
-    this.expiresIn = expiresIn;
-    this.refreshToken = refreshToken;
-  }
-  
+
+	private static final long serialVersionUID = 1L;
```
Comment (UglyTroLL) [inline src/main/java/com/dongxuexidu/douban4j/model/app/AccessToken.java]: Please keep the original format for this project (the indent was changed here, which is supposed to be 2)

File: src/main/java/com/dongxuexidu/douban4j/model/app/DoubanException.java [hunk 2]
```diff
@@ -5,42 +5,46 @@
...
-    this.errorMsg = errorMsg;
-  }
-  
-  public DoubanException (int code, String msg) {
-    super(msg);
-    this.errorCode = code;
-    this.errorMsg = msg;
-  }
-  
+
```
Comment (UglyTroLL) [inline src/main/java/com/dongxuexidu/douban4j/model/app/DoubanException.java]: same comments as the previous file

File: src/main/java/com/dongxuexidu/douban4j/playground/PlayGround.java [hunk 3]
```diff
@@ -117,7 +116,7 @@ public static void testSendingDoumail () {
   
   public static void testGetBookInfo () {
     try {
-      long bookId = 2023013;
+      // long bookId = 2023013;
```
Comment (UglyTroLL) [inline src/main/java/com/dongxuexidu/douban4j/playground/PlayGround.java]: why?

**Comment by UglyTroLL** [comment]
Thanks a lot for doing this

File: src/main/java/com/dongxuexidu/douban4j/model/app/AccessToken.java (hunk 1 above)
Comment (effine) [inline src/main/java/com/dongxuexidu/douban4j/model/app/AccessToken.java]: ok !

File: src/main/java/com/dongxuexidu/douban4j/playground/PlayGround.java (hunk 3 above)
Comment (effine) [inline src/main/java/com/dongxuexidu/douban4j/playground/PlayGround.java]: the bookId is not used !

**Comment by effine** [comment]
standard format commit

**Comment by effine** [comment]
修正格式之后提交了！

**Pull request merged**